### DISCUSSION TITLE
What's the preferred way to access a get-only property on an item?

### DISCUSSION THREAD

**Discussion by JOyo246** [pointfreeco/swift-identified-collections#67] [Q&A]
There is a WritableKeyPath accessor to grab a property on value, but that won't work if the property is a constant.

So, it seems like we have to do, `someIdentified.value.someProperty`, is that the best way?

Is there a reason why something like this does not exist?

```
extension Identified {
    public subscript<Subject>(
      dynamicMember keyPath: KeyPath<Value, Subject>
    ) -> Subject {
        self.value[keyPath: keyPath]
    }
}
```

**Comment by mbrandonw** [comment]
Hi @JOyo246, this is a good catch! Would you be interested in PR'ing a fix for it?

**Reply by JOyo246** [reply]
My first PR on a Point-Free repo! How does this look? #68
